src/docs.c

```diff
@@ -41,19 +41,25 @@
 static int usage (struct GMTAPI_CTRL *API, int level) {
 	const char *name = gmt_show_name_and_purpose (API, THIS_MODULE_LIB, THIS_MODULE_CLASSIC_NAME, THIS_MODULE_PURPOSE);
 	if (level == GMT_MODULE_PURPOSE) return (GMT_NOERROR);
-	GMT_Message (API, GMT_TIME_NONE, "usage: %s [-Q] [-S] [%s] <module-name> [<-option>]\n\n", name, GMT_V_OPT);
+	GMT_Usage (API, 0, "usage: %s [-Q] [-S] [%s] <module-name> [<-option>]\n\n", name, GMT_V_OPT);
 
 	if (level == GMT_SYNOPSIS) return (GMT_MODULE_SYNOPSIS);
-	GMT_Message (API, GMT_TIME_NONE, "\t<module-name> is one of the core or supplemental modules, or one of\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   api, colors, cookbook, data, gallery, gmt, home, settings, and tutorial.\n");
-	GMT_Message (API, GMT_TIME_NONE, "\t   Also acceptable are forum (GMT Discourse Forum) and [web]site (GMT Main Website).\n");
+
+	GMT_Message (API, GMT_TIME_NONE, "  REQUIRED ARGUMENTS:\n");
+	GMT_Usage (API, 1, "\n<module-name>");
+	GMT_Usage (API, -2, "One of the core or supplemental modules, or one of "
+		"api, colors, cookbook, data, gallery, gmt, home, settings, and tutorial. "
+		"Also acceptable are forum (GMT Discourse Forum) and [web]site (GMT Main Website).");
```

Comment:
```suggestion
		"Also acceptable are forum (GMT Discourse Forum) and website (GMT Main Website).");
```
`gmt docs web` does not work but `gmt docs website` does

Reply:
Carried over from original. `gmt docs site` works as well. 

Maybe shift the parentheses? -> `web[site]`

Reply:
> Carried over from original. `gmt docs site` works as well.
> 
> Maybe shift the parentheses? -> `web[site]`

sounds good

Reply:
```suggestion
		"Also acceptable are forum (GMT Discourse Forum) and web[site] (GMT Main Website).");
```